editor/src/components/editor/canvas-toolbar.spec.browser2.tsx

```diff
@@ -623,6 +623,107 @@ describe('canvas toolbar', () => {
     )
   })
 
+  it('can insert an array via the floating insert menu', async () => {
+    const editor = await renderTestEditorWithCode(
+      makeTestProjectCodeWithComponentInnards(`
+      const myArray = ['one', 'two', 'three']
+      return (
+        <div
+        style={{
+          backgroundColor: '#aaaaaa33',
+          position: 'absolute',
+          left: 57,
+          top: 168,
+          width: 247,
+          height: 402,
+        }}
+        data-uid='container'
+      >
+        <div data-uid='a3d' />
+      </div>
+    )`),
+      'await-first-dom-report',
+    )
+
+    await selectComponentsForTest(editor, [
+      EP.fromString(`${BakedInStoryboardUID}/${TestSceneUID}/${TestAppUID}:container`),
+    ])
+
+    await insertViaAddElementPopup(editor, 'myArra')
+
+    expect(getPrintedUiJsCode(editor.getEditorState())).toEqual(
+      makeTestProjectCodeWithComponentInnards(`
+      const myArray = ['one', 'two', 'three']
+      return (
+        <div
+        style={{
+          backgroundColor: '#aaaaaa33',
+          position: 'absolute',
+          left: 57,
+          top: 168,
+          width: 247,
+          height: 402,
+        }}
+        data-uid='container'
+      >
+        <div data-uid='a3d' />
+        <React.Fragment>{myArray.map((item) => (<span>{JSON.stringify(item)}</span>))}</React.Fragment>
+      </div>
+    )`),
+    )
+  })
+
+  it.skip('can insert an image via the floating insert menu', async () => {
```

Comment:
Why the skipped test?

Reply:
oh nice catch!

Reply:
<img width="382" alt="Screenshot 2023-11-22 at 2 54 05" src="https://github.com/concrete-utopia/utopia/assets/7003853/ca154aa7-5c1e-4d9a-a56f-0b5433c2cb47">
Still fighting with it :) (strings look identical but test fails. Tried copying the "actual" to the "expected", no luck)